test-app/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:lts-iron as builder
```

Comment:
Wondering what the best location for this file would be, as it is typically run in the context of the monorepo root directory, and not in the context of the `test-app` package.

Reply:
It's true that it does need to be run in the monorepo context, otherwise it doesn't have all the files it needs. I put (left? I forget) it here as it relates only to the test app. I doubt we'll want a 'real' Dockerfile at any point in the future though, so it probably could just be in the root. I'm not sure of any reason to do one over the other.

Reply:
Yeah, I don't really have an opinion either, but it just makes more sense to me that the dockerfile is placed at the location where it is built.

Reply:
We can of course also add a comment to the `Dockerfile` explaining that you need to run it in the context of the monorepo root. In the `docker-compose` file, we can set the `context` attribute to `..`

Reply:
If we leave the `docker`-related files in the `test-app` package, `docker-compose.yml` would have the following structure:
```yaml
version: "3.8"
services:
  test-app:
    build:
      context: ..
      dockerfile: ./test-app/Dockerfile
    ports:
      - "80:80"
   ```

Reply:
I'm now also wondering if it is really necessary to keep the compose file in this repo, as it is quite straightforward.

Reply:
It's very straightforward, but I think it makes sense to keep it as otherwise we have to maintain an unversioned file on the server.